doc-content/enterprise-only/ai/ai-credit-card-con.adoc

```diff
@@ -43,7 +43,7 @@ The *Diagram* tab contains the BPMN diagram that shows the decision paths that t
 
 image:ai/bpmn.png[]
 
-The *Decision Task* task contains the rules that determine whether the dispute will be automatically charged back to Joe's account (*Milestone 1*) or whether more investigation is required (*Milestone 2*) based on the values of the `caseFile_cardholderRiskRating` and the `caseFile_disputeRiskRating` variables. If Joe's dispute matches the criteria for automatic approval, Milestone 1 is followed and the disputed amount is charged back to his account. If Joe's dispute requires manual evaluation, the process in Milestone 2 is started.
+The *Decision Task* task contains the rules that determine whether the dispute will be automatically charged back to Joe's account (*Milestone 1*) or whether more investigation is required (*Milestone 2*) based on the values of the `caseFile_cardholderRiskRating` and the `caseFile_disputeRiskRating` variables. If Joe's dispute matches the criteria for automatic approval, Milestone 1 is followed and the disputed amount is charged back to his account. This sub-process is very lean and efficient. If Joe's dispute requires manual evaluation, the process in Milestone 2 is started, which will require some human actor involvement and will take more resources to process.
```

Comment:
"automatic approval, Milestone 1 is followed" Shouldn't the milestone be tagged as *Milestone 1* ?

Reply:
Well, I'll go with that suggestion. But my thinking was that in the first instance I was directly referring to the UI, But the ones you are referring to I was meaning the process. But in this case they really are one in the same. Make sense?Anyway, I changed

Reply:
I do understand what you are saying. However, it just seemed like a UI mention to me.